rosalind_hamm.py

```diff
@@ -0,0 +1,9 @@
+def main(data):
+    a, b = data[:-1].split('\n')
```

Comment:
The hamm distance of a string will be the same whether it is reversed or not.

There is no reason to reverse the string here.